addon-test-support/should-force-audit.ts

```diff
@@ -1,17 +1,40 @@
 import { ENABLE_A11Y_AUDIT } from './cli-options';
 
-export function setEnableA11yAudit(enabled: boolean = false) {
-  const url = new URL(window.location.href, document.baseURI);
+const A11Y_AUDIT_PARAM = 'enableA11yAudit';
```

Comment:
We could also just inline this, since it makes it easier to read 🤷‍♂️

Reply:
Yeah, there was more usage before I refactored. Thanks for the fresh eyes!